rust/src/wallet.rs

```diff
@@ -362,7 +364,7 @@ pub async fn open_channel(peer_info: PeerInfo, taker_amount: u64) -> Result<()>
 
     let address_to_fund = get_address()?;
 
-    let client = reqwest::Client::new();
+    let client = reqwest::Client::builder().timeout(TCP_TIMEOUT).build()?;
```

Comment:
We should probably also do that for get offer.

Reply:
happy to accept a PR for that :)

Reply:
I was starting with it, but I was wondering how we want to continue with our constants? For now I can use it from the wallet.rs, but I don't think it should stay there.

Reply:
https://github.com/itchysats/10101/pull/249